temporian/core/operators/calendar/base.py

```diff
@@ -56,9 +66,32 @@ def build_op_definition(cls) -> pb.OperatorDef:
         return pb.OperatorDef(
             key=cls.operator_def_key(),
             inputs=[pb.OperatorDef.Input(key="sampling")],
+            attributes=[
+                pb.OperatorDef.Attribute(
+                    key="utc_offset",
+                    type=pb.OperatorDef.Attribute.Type.FLOAT_64,
+                ),
+            ],
             outputs=[pb.OperatorDef.Output(key="output")],
         )
 
+    @staticmethod
+    def convert_to_utc_offset(timezone: Union[str, float, int]) -> float:
```

Comment:
would de-attach this method from this class since it doesn't win anything by being in it (could be here outside the class, or in dtype_normalization, or somewhere else)